tests/UI/campaigns/functional/BO/00_login/03_logout.ts

```diff
@@ -0,0 +1,50 @@
+// Import utils
+import helper from '@utils/helpers';
+import mailHelper from '@utils/mailHelper';
+
+// Import test context
+import testContext from '@utils/testContext';
```

Comment:
```suggestion
import helper from '@utils/helpers';
import mailHelper from '@utils/mailHelper';
import testContext from '@utils/testContext';
```

Reply:
Franck,
When I change this line to :  
    import helper2 from '@utils/helpers';
I got this message: "Cannot find module '@utils/helpers' or its corresponding type declarations.ts"

But when I change this line to using the full path : 
    import helper from '/var/www/html/PrestaShop/tests/UI/utils/helpers';
It seems OK ! An explanation?